docs/deployment_patterns/how_to_run_a_checkpoint_in_airflow.md

```diff
@@ -14,59 +14,254 @@ This guide will help you run a Great Expectations checkpoint in Apache Airflow,
+
+```python
+checkpoint_config = CheckpointConfig(
+    **{
+        "name": "taxi.pass.chk",
+        "config_version": 1.0,
+        "template_name": None,
+        "module_name": "great_expectations.checkpoint",
+        "class_name": "Checkpoint",
+        "run_name_template": "%Y%m%d-%H%M%S-my-run-name-template",
+        "expectation_suite_name": "taxi.demo",
+        "batch_request": None,
+        "action_list": [
+            {
+                "name": "store_validation_result",
+                "action": {"class_name": "StoreValidationResultAction"},
+            },
+            {
+                "name": "store_evaluation_params",
+                "action": {"class_name": "StoreEvaluationParametersAction"},
+            },
+            {
+                "name": "update_data_docs",
+                "action": {"class_name": "UpdateDataDocsAction", "site_names": []},
+            },
+        ],
+        "evaluation_parameters": {},
+        "runtime_configuration": {},
+        "validations": [
+            {
+                "batch_request": {
+                    "datasource_name": "my_datasource",
+                    "data_connector_name": "default_inferred_data_connector_name",
+                    "data_asset_name": "yellow_tripdata_sample_2019-01.csv",
+                    "data_connector_query": {"index": -1},
+                },
+            }
+        ],
+        "profilers": [],
+        "ge_cloud_id": None,
+        "expectation_suite_ge_cloud_id": None,
+    }
+)
+```
 
-Another option is to use the output of the `great_expectations checkpoint script` command and paste it into a method that is called from a PythonOperator in the DAG. This gives you more fine-grained control over how to respond to Validation Results:
+The `DataContextConfig` and  `CheckpointConfig` objects may be defined in the DAG file, but to avoid excess top-level code, it is recommended these objects be defined in a config file in the `include/` directory and imported into the DAG file.
 
-1. Run `great_expectations checkpoint script`
+For a full list of parameters, see the `GreatExpectationsOperator` [documentation](https://registry.astronomer.io/providers/great-expectations/modules/greatexpectationsoperator). Additional [Great Expectations example DAGs](https://registry.astronomer.io/dags?providers=Great+Expectations&page=1) may be found on the [Astronomer Registry](https://registry.astronomer.io/).
 
-    ```bash
-    great_expectations checkpoint script my_checkpoint
+## Connections
```

Comment:
This seems like it's more about configuring backends than the Airflow connection.

Reply:
I could change the title to `Connections and Backends`, because really both are needed.